uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request.controller.js

```diff
@@ -12,23 +12,25 @@ define([
   'common/services/pub-sub',
   'leave-absences/shared/models/absence-period-model',
   'leave-absences/shared/models/absence-type-model',
-  'leave-absences/shared/models/calendar-model',
   'leave-absences/shared/models/entitlement-model',
   'leave-absences/shared/models/leave-request-model',
-  'leave-absences/shared/models/public-holiday-model'
-], function (angular, controllers, _, moment) {
+  'leave-absences/shared/models/public-holiday-model',
+  'leave-absences/shared/instances/leave-request.instance',
+  'leave-absences/shared/instances/sickness-request.instance',
+  'leave-absences/shared/instances/toil-request.instance'
+], function (angular, controllers, _) {
   'use strict';
 
   controllers.controller('RequestCtrl', RequestCtrl);
 
-  RequestCtrl.$inject = ['$log', '$q', '$rootScope', 'Contact', 'dialog',
-    'AbsencePeriod', 'AbsenceType', 'api.optionGroup', 'checkPermissions',
-    'pubSub', 'Calendar', 'Entitlement', 'HR_settings', 'Session',
-    'LeaveRequest', 'PublicHoliday', 'shared-settings'];
+  RequestCtrl.$inject = [
```

Comment:
Tbh I quite like the previous syntax. It is more compact, it matches max 80 symbols per line and I don't see any advantage of the syntax it was changed to.